src/gui/guiFormSpecMenu.cpp

```diff
@@ -907,8 +886,8 @@ void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &el
 
 	core::rect<s32> rect = core::rect<s32>(pos, pos + geom);
 
-	GUIAnimatedImage *e = new GUIAnimatedImage(Environment, this, spec.fid,
-		rect, texture_name, frame_count, frame_duration, m_tsrc);
+	irr_ptr<GUIAnimatedImage> e(new GUIAnimatedImage(Environment, data->current_parent,
```

Comment:
Changed behaviour - is this a bugfix?

Reply:
Yes. Changing the parent for this element was probably forgotten in a rebase.